RELEASE/scripts/autoscend/auto_quest_level_8.ash

```diff
@@ -371,11 +552,10 @@ boolean L8_trapperGroar()
 			if (item_amount($item[Groar\'s Fur]) == 0 && item_amount($item[Winged Yeti Fur]) == 0)
 			{
 				addToMaximize("2000cold resistance 5max");
-				// mafia bug screws us here if we don't already have the 5 cold res. auto_pre_adv will not be called and this will infinite loop
-				equipMaximizedGear(); // remove this once https://kolmafia.us/showthread.php?24764-betweenBattleScript-and-afterAdventureScrinever-gets-called is fixed.
 				//If this returns false, we might have finished already, can we check this?
-				autoAdv(1, $location[Mist-shrouded Peak]);
+				boolean retval = autoAdv(1, $location[Mist-shrouded Peak]);
 				removeFromMaximize("2000cold resistance 5max");
```

Comment:
I'm a bit confused why we remove from the maximizer here - wouldn't it get reset by `resetMaximize()` in `doTasks()`? But not relevant to this PR.

Reply:
I made that change too when trying to fix the issue where mafia wouldn't allow us to adventure at the peak without 5 resistance already equipped (it checks and just blocks you so it doesn't even run your betweenBattleScript which is where we would fix it).

I'll remove it as you're correct, it's not necessary.